src/main/java/use_cases/add_edit_category_use_case/CategoryOB.java

```diff
@@ -5,23 +5,21 @@
  */
```

Comment:
Why is this Javadoc formatted in pothole case? make sure all of your Javadocs are in camel case.